Modules/Microsoft365DSC/DSCResources/MSFT_AADRoleSetting/MSFT_AADRoleSetting.psm1

```diff
@@ -1253,7 +1312,6 @@ function Export-TargetResource
                 ApplicationId         = $ApplicationId
                 TenantId              = $TenantId
                 CertificateThumbprint = $CertificateThumbprint
-                Credential            = $Credential
```

Comment:
Why are we removing credentials from here?

Reply:
I think this was a mistake, i have corrected it in the next commit, Thanks.